apps/hub-app/src/components/DaoCard.tsx

```diff
@@ -92,9 +93,9 @@ export const DaoCard = ({
         <Tag>{networkId}</Tag>
         <Tag>{contractType}</Tag>
       </div>
-      <div>
+      {/* <div>
         <Button secondary>Go</Button>
-      </div>
+      </div> */}
```

Comment:
Delete? Add a comment if you want to keep for future?

Reply:
Do want to keep for future. Good catch though